internal/generator/vector/inputs_to_pipeline.go

```diff
@@ -0,0 +1,37 @@
+package vector
+
+import (
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	. "github.com/openshift/cluster-logging-operator/internal/generator"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/helpers"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/source"
+	transform "github.com/openshift/cluster-logging-operator/internal/generator/vector/transform"
+)
+
+
+func InputsToPipeline(spec *logging.ClusterLogForwarderSpec, op Options, sources []source.LogSource) []transform.Transform {
+ transformerList := []transform.Transform{}
+ var instance transform.Transform
+
+ for _, source := range sources{
+
+   if source.Type() == "journald" {
```

Comment:
better to do this matching on CLF spec rather than the config we are trying to produce